contents/tutorials/nps-survey.md

```diff
@@ -0,0 +1,102 @@
+
+To do this, enter ‘How likely are you to recommend this product to a friend?’ in the Question field. Leave the optional description blank. 
+
+NPS surveys collect responses on a scale of 1-10, so set the Question Type to ‘Rating’. Change the Display Type to ‘Number’, and the Scale to ‘1 - 10’. 
+
+We want to make sure that users definitely understand what each number means, so set the Lower Bound Label to ‘Unlikely’ and the Upper Bound Label to ‘Likely’. You should now see a complete NPS survey in the live preview, and can tailor the appearance to match your brand. 
+
+![Create an NPS survey](../images/tutorials/nps-survey/nps_survey_1.png)
+
+### Targeting your NPS survey
+
+The default behaviour for PostHog surveys is to target all users, but you can choose to override this and target users based on the URL they are on, user properties, or using a particular feature flag. 
+
+In order to collect a valid result you really need at least 100 responses, so it’s important not to target too narrowly. However, you _also_ want to target users who are familiar with your product in order to get a meaningful result. Good options for striking a balance include…
+
+- Using ‘URL contains’ to target users who view a particular page, such as a checkout confirmation page. 
+- Using ‘User properties’ to target users in a particular geographical area, or in a dynamic cohort of users that have performed an action (such as completing a purchase).
+- Using a linked feature flag to target users who have been trialing a new feature or UI rollout
+
+If you set targeting options for your survey then you’ll also need to decide the release conditions within that target. That is: what percentage of that defined audience will see the survey? The default is 0%, so make sure to set it to a higher value. 
+
+Next, you can save your survey and press Launch to start collecting responses. 
+
+### Calculating your NPS score in PostHog
+Once you’ve launched your survey, the next step is to wait for some responses. After that, it’s time to analyze the results!
+
+Start by taking note of the survey ID for your NPS survey, which you can find in the URL of the results page.
+
+![PostHog survey ID](../images/tutorials/nps-survey/nps_survey_2.png)
+
+> **Can't I use `Survey Name` instead?** Technically, yes. However because it's possible for other users to create surveys with similar names, or to rename your survey, the survey ID is generally a more stable option.
+
+Next, head to the Insights section and create a new Trends insight. Set the first series event as ‘Survey sent’ and create a filter to specify the Survey ID. Duplicate this series three times.
+
+![PostHog NPS filters](../images/tutorials/nps-survey/nps_survey_3.png)
+
+Next, we start grouping results into Detractors, Passives and Promoters. For the first series, add an additional filter which uses the Survey Response event and filters so that that event is equal to 1, 2, 3, 4, 5, and 6. This is your Detractor cohort. 
```

Comment:
```suggestion
Next, we start grouping results into Detractors, Passives and Promoters. For the first series, add an additional filter which uses the 'Survey Response' event and filters so that that event is equal to 1, 2, 3, 4, 5, and 6. This is your Detractor cohort. 
```